addon/components/paper-progress-linear.js

```diff
@@ -1,5 +1,7 @@
 import Ember from 'ember';
 import ColorMixin from 'ember-paper/mixins/color-mixin';
+const {inject, computed, Component, isPresent} = Ember;
```

Comment:
to conform with other stlyes, please add spaces around declarations.

`const { inject, computed, Component, isPresent } = Ember;`

Sorry for this. We need to add decent jscs support in the project.